plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -0,0 +1,945 @@
+
+        if ap_config.get("ap_name"):
+            param_spec = dict(type = "str", length_max = 32)
+            validate_str(ap_config["ap_name"], param_spec, "ap_name", errormsg)
+
+        if ap_config.get("ap_name_new"):
+            param_spec = dict(type = "str", length_max = 32)
+            validate_str(ap_config["ap_name_new"], param_spec, "ap_name_new", errormsg)
+
+        if ap_config.get("led_brightness_level"):
+            if ap_config["led_brightness_level"] not in range(1,9):
+                errormsg.append("led_brightness_level: Invalid LED Brightness level '{0}' in playbook"\
+                                .format(ap_config["led_brightness_level"]))
+
+        if ap_config.get("led_status") and ap_config.get("led_status") not in ("Disabled", "Enabled"):
+            errormsg.append("led_status: Invalid LED Status '{0}' in playbook"\
+                            .format(ap_config["led_status"]))
+
+        if ap_config.get("location"):
+            param_spec = dict(type = "str", length_max = 255)
+            validate_str(ap_config["location"], param_spec, "location",
+                            errormsg)
+
+        if len(errormsg) > 0:
+            self.msg = "Invalid parameters in playbook config: '{0}' "\
+                     .format(str("\n".join(errormsg)))
+            self.log(self.msg, "ERROR")
+            self.status = "failed"
+            return self
+
+        # check if the given AP config exists and/or needs to be updated/created.
+        if self.have.get("ap_exists"):
+            consolidated_data = self.compare_ap_config_with_inputdata(self.have["current_ap_config"])
+
+            if not consolidated_data:
+                # Accesspoint does not need update
+                self.msg = "AP - {0} does not need any update"\
+                    .format(self.have.get("current_ap_config").get("ap_name"))
+                self.log(self.msg, "INFO")
+                responses = {}
+                del self.payload["device_list"]
+                responses["accesspoints_updates"] = {"response": self.payload["device_config"]}
+                self.result['msg'] = self.msg
+                self.result["response"].append(responses)
+                self.result["changed"] = False
+                return self
+
+            self.log('Final AP Configuration data to update {0}'.format(str(consolidated_data)),
+                      "INFO")
+            task_response = self.update_ap_configuration(consolidated_data)
+            self.log('Task respoonse {0}'.format(str(task_response)),"INFO")
+            config_updated = True
+
+        responses = {}
+        if config_updated or config_created:
+            if task_response and isinstance(task_response, dict):
+                self.check_task_response_status(task_response, "task_intent", True)\
+                    .check_return_status()
+                self.log("Status of the task is {}.".format(self.status), "INFO")
```

Comment:
Don't need to print the status if it became failed then we actually stopped the task execution otherwise y default it's success only.